Entries/Copy-X/Copy-X.md

```diff
@@ -0,0 +1,336 @@
+
+#### Unconfirmed: 
+
+	*copy-X (Unsure which variant):
+		* Alberts mysterioese Insel (Germany)[^116827]
+		* Ein Fall fuer TKKG - Das geheimnisvolle Testament (Germany)[^108587]
+		* Onkel Alberts geheimnisvolles Notizbuch (Germany)[^116828]
+		* Webmaster (Germany)[^116418]
+	*copy-X light:
+		* German Westerner 2[^pcgames_de_westerner2][^gamestar_de_westerner2]
+	*copy-X audio:
+		* Die Wilden Kerle – DWK4 - Der Angriff Der Silberlichten (Das Hörspiel Zum Kinofilm)[^D_8232013]
+		* Laava – Wherever You Are (I Feel Love)[^D_1436646]
+		* Manfred Mann* – 2006[^D_2182905]
+		* De-Phazz – Godsdog[^D_451419]
+
+* Many games from Sixteen Tons Entertainment
+* Many games from Tivola
+* Many games from Terzio
+* Many pieces of software from Magix 
+
+[^116827]: [Alberts mysterioese Insel (Germany)](http://redump.org/disc/116827/)
+[^108587]: [Ein Fall fuer TKKG - Das geheimnisvolle Testament (Germany)](http://redump.org/disc/108587/)
+[^116828]: [Onkel Alberts geheimnisvolles Notizbuch (Germany)](http://redump.org/disc/116828/)
+[^116418]: [Webmaster (Germany)](http://redump.org/disc/116418/)
+[^D_8232013]: [Die Wilden Kerle – DWK4 - Der Angriff Der Silberlichten (Das Hörspiel Zum Kinofilm)](https://www.discogs.com/release/8232013-Die-Wilden-Kerle-DWK4-Der-Angriff-Der-Silberlichten-Das-H%C3%B6rspiel-Zum-Kinofilm)
+[^D_1436646]: [Laava – Wherever You Are (I Feel Love)](https://www.discogs.com/release/1436646-Laava-Wherever-You-Are-I-Feel-Love)
+[^D_2182905]: [Manfred Mann* – 2006](https://www.discogs.com/release/2182905-Manfred-Mann-2006)
+[^D_451419]: [De-Phazz – Godsdog](https://www.discogs.com/release/451419-De-Phazz-Godsdog)
+[^pcgames_de_westerner2]: [The Westerner 2 im PC-Games-Test: "Ein enttäuschendes Adventure ... " ](https://web.archive.org/web/20090322110537/http://www.pcgames.de/aid,679282/The-Westerner-2-im-PC-Games-Test-Ein-enttaeuschendes-Adventure-/PC/Test/)
+[^gamestar_de_westerner2]: ["The Westerner 2" (gamestar.de Publication) (Archive pending)](https://download.gamestar.de/public/43400/43452/05_2009_096.pdf)
+
+***
+
+## Supported Platforms:
+
+* copy-X light: Any (no software protection is employed, although it appears to mainly be marketed for Windows games regardless.)
+
+* copy-X professional: Windows
+(Some professional games are Hybrid PC/Mac discs, but the Mac versions of these games do not appear to utilize any sort of software protection.)[^108150]
+
+* copy-X audio: Audio CD
+
+***
+
+## Software Known to Detect:
+
+Note: While ProtectionID does detect copy-X, it detects it as "Optgraph Copy-X / Ring-Protech". As far as current research has found, there is no relation between copy-X and Ring Protech other than both being ring-based protections, and the optgraph file in question is not found on Ring Protech. PiD is thus believed to be incorrect in designating it also Ring-Protech, and it's just detecting Copy-X in these instances.
```

Comment:
Because the note is specifically about PiD, this would be better integrated into the line for ProtectionID in some way. For example, something like:
`` * ProtectionID[^PiD_List] (Note: ...)``